src/monthlychallenges/nov-2020/nov-2020.json

```diff
@@ -51,6 +51,11 @@
           "url": "https://dev.to/jarvisscript/connecting-now-swag-2e9i",
           "count": 412
         }
+        {
```

Comment:
Thanks Chris! Looks like we need a comma here and then we can get this merged in!